mne/io/reference.py

```diff
@@ -193,6 +193,12 @@ def add_reference_channels(inst, ref_channels, copy=True):
                      'unit': FIFF.FIFF_UNIT_V,
                      'coord_frame': FIFF.FIFFV_COORD_HEAD,
                      'loc': np.zeros(12)}
+        # see if "zeroth" EEG electrode present in dig points
+        dig_loc = [dl for dl in inst.info['dig'] if (
+                   dl['kind'] == FIFF.FIFFV_POINT_EEG and
+                   dl['ident'] == 0)]
+        if len(dig_loc) == 1:
+            chan_info['loc'][:3] = dig_loc[0]['r'][:]
```

Comment:
...and actually if `len(dig_loc) > 1` I wonder if we should set the `info['custom_ref_applied'] = True` to prevent people from errantly doing source localization